openstack-installer/roles/mongodb/tasks/main.yml

```diff
@@ -46,6 +46,12 @@
   run_once: True
   when: mongo_initialized == False
   until: mongo_status.stdout == '1'
+  retries: 10
+  delay: 5
+  # FIXME: Ansible 2.0 does not fail when the task has retried for 'retries'
+  # and the 'until' condition is not met.
+  # https://github.com/ansible/ansible/issues/14461
+  failed_when: mongo_status.stdout != '1'
 
```

Comment:
The retries and delay lines are necessary as the task would fail on a slower computer (eg. on my home computer I tried)